synapse/lib/ast.py

```diff
@@ -2478,6 +2480,10 @@ async def expr_or(x, y):
     return await tobool(x) or await tobool(y)
 async def expr_and(x, y):
     return await tobool(x) and await tobool(y)
+async def expr_re(x, y):
+    if regex.search(await tostr(y), await tostr(x), flags=regex.IGNORECASE):
```

Comment:
Would we also want to do multiline here as well?

Reply:
Probably whatever makes it most "grep like" so, we may wanna enable multi-line evaluation.  We'll probably eventually have legit regex instances and such.  May not want IGNORECASE as the default.